rs/execution_environment/src/lib.rs

```diff
@@ -98,21 +99,27 @@ impl ExecutionServices {
         metrics_registry: &MetricsRegistry,
         own_subnet_id: SubnetId,
         own_subnet_type: SubnetType,
-        scheduler_config: SchedulerConfig,
         config: Config,
-        cycles_account_manager: Arc<CyclesAccountManager>,
+        subnet_config: SubnetConfig,
         state_reader: Arc<dyn StateReader<State = ReplicatedState>>,
         fd_factory: Arc<dyn PageAllocatorFileDescriptor>,
         completed_execution_messages_tx: Sender<(MessageId, Height)>,
         temp_dir: &Path,
     ) -> ExecutionServices {
+        let cycles_account_manager = Arc::new(CyclesAccountManager::new(
+            subnet_config.scheduler_config.max_instructions_per_message,
+            own_subnet_type,
+            own_subnet_id,
+            subnet_config.cycles_account_manager_config,
+        ));
+
```

Comment:
NIT: maybe have a separate variable reference to scheduler_config: SchedulerConfig, so that other lines stay short.